src/openforms/contrib/customer_interactions/update.py

```diff
@@ -72,30 +89,61 @@ def update_customer_interaction_data(
 
     channels_to_address_types = {v: k for k, v in ADDRESS_TYPES_TO_CHANNELS.items()}
 
+    result: UpdateCustomerInteractionsResult = {}
```

Comment:
How does this type check if the keys are `Required` by default 🤔 ?

Reply:
included it into `pyright.pyproject.toml`